kythe/release/appengine/buildbot/cloudbuild.yaml

```diff
@@ -0,0 +1,29 @@
+steps:
+- name: gcr.io/cloud-builders/gsutil
+  id: 'Fetch secrets'
+  args:
+  - cp
+  - gs://kythe-buildbot/secrets.tar.enc
+  - secrets.tar.enc
+- name: gcr.io/cloud-builders/gcloud
+  id: 'Decrypt secrets'
+  args:
+  - kms
```

Comment:
maybe put a comment above this step pointing to the gcb doc on kms

https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-secrets-credentials

Reply:
Done.